src/styles/tickets/frontend.pcss

```diff
@@ -3,308 +3,339 @@
  *
  * @since  0.3.0-alpha
  */
-.tribe-block__tickets {
+.tribe-common.tribe-block__tickets {
+	border: 1px solid var(--color-border-secondary);
+	border-radius: 4px;
 	margin-top: 30px;
 	margin-bottom: 30px;
+	padding: 1rem;
```

Comment:
rem. you can also use the spacer variables

Reply:
Yeah, these are going away - ignore this file.